### PR TITLE
Front-load core devs in citation.cff, sorted by commits

### PR DESCRIPTION
**Pull request by jni**
This was discussed in the [core-devs Zulip chat](https://napari.zulipchat.com/#narrow/stream/320379-core-devs/topic/authorship.20order.2C.20paper/near/437218905) (core dev permissions needed). We didn't quite get consensus and all solutions are flawed, because authorship order is trying to compress a complex and multidimensional credit system into a not-even-continuous 1D variable, but this at least restores the status quo from before the last update and is a *reasonable* approximation of the overall contributions to the project.

### CODE REVIEW

**Comment by jni** [comment]
@jules-vanaret I just noticed you don't have an entry and you definitely should have! Could you please fill in this template:

```yaml
- given-names:
  family-names:
  affiliation:
  orcid: (optional)
  alias: (gh handle)
```

Then I can add you in this same PR.

File: CITATION.cff [hunk 1]
```diff
@@ -5,6 +5,79 @@ identifiers:
...
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
+  orcid: https://orcid.org/0000-0002-5470-1676
+  alias: Czaki
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Monash eResearch Centre, Monash University
+  alias: jni
+- given-names: Peter
```
Comment (psobolewskiPhD) [inline CITATION.cff:27]: @jni How could I possibly be in this slot if it's by commit?

Comment (jni) [reply CITATION.cff:27]: I added together this repo and docs. 😃

File: CITATION.cff [hunk 2]
```diff
@@ -93,11 +136,6 @@ authors:
   affiliation: Ramona Optics Inc, Durham, North Carolina, USA
   orcid: https://orcid.org/0000-0002-4657-4603
   alias: hmaarrfk
-- given-names: Kyle I. S.
```
Comment (psobolewskiPhD) [inline CITATION.cff:96]: @jni you cut Kyle here, but I don't see Kyle pasted back anywhere.

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]

File: CITATION.cff (hunk 2 above)
Comment (jni) [reply CITATION.cff:96]: gdi. Anyway, code review for the win! 😅 Thanks!

**Comment by jules-vanaret** [comment]
> @jules-vanaret I just noticed you don't have an entry and you definitely should have! Could you please fill in this template:
> Then I can add you in this same PR.

Wow thank you very much @jni, that means a lot!
Here's my entry:
```yaml
- given-names: Jules
  family-names: Vanaret
  affiliation: Aix Marseille University, CNRS, Fresnel, I2M, IBDM, Turing Centre for Living systems
  orcid: https://orcid.org/0009-0004-6070-2263
  alias: jules-vanaret
 ```

File: CITATION.cff [hunk 3]
```diff
@@ -5,6 +5,84 @@ identifiers:
...
+  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
+  orcid: https://orcid.org/0000-0002-5470-1676
+  alias: Czaki
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Monash eResearch Centre, Monash University
+  alias: jni
+- given-names: Peter
+  family-names: Sobolewski
+  affiliation: West Pomeranian University of Technology, Szczecin
```
Comment (psobolewskiPhD) [inline CITATION.cff]:
```suggestion
  affiliation: The Jackson Laboratory
```
Thanks for the reminder @Czaki

File: CITATION.cff (hunk 1 above)
Comment (Czaki) [reply CITATION.cff:27]: @jni Did you do this manually, or you created a script to automatize this before each release?

File: CITATION.cff [hunk 4]
```diff
@@ -5,6 +5,84 @@ identifiers:
...
+  family-names: Royer
+  affiliation: Chan Zuckerberg Biohub
+  alias: royerloic
+- given-names: Ahmet
+  family-names: Can Solak
+  affiliation: Chan Zuckerberg Biohub
+  alias: AhmetCanSolak
+- given-names: Kyle I. S.
+  family-names: Harrington
+  affiliation: Chan Zuckerberg Initiative
```
Comment (psobolewskiPhD) [inline CITATION.cff:83]: @kephale Does this need to be updated to CZII?

Comment (kephale) [reply CITATION.cff:83]: Long story, but it does not need to be updated. Thanks for checking 🙏

File: CITATION.cff (hunk 1 above)
Comment (jni) [reply CITATION.cff:27]: Manually. 😬

**Review by psobolewskiPhD — APPROVED** [review]

File: CITATION.cff (hunk 1 above)
Comment (Czaki) [reply CITATION.cff:27]: Ah. 
In general, GitHub GUI counts also coauthor ship of commits, that is not easy accessible from API. 

@psobolewskiPhD it looks like @jni used data where being coauthors of PR (having accepted suggestion to merged PR) is counting, so your activity in reviewing PR and suggesting updates boost your position.

**Review by Czaki — APPROVED** [review]

**Comment by jni** [comment]
I'm gonna merge this to be able to have a new PR in which I add all the missing folks since 0.4.19. Thanks all for review!

**Pull request merged**